ui/pages/confirmations/selectors/confirm.ts

```diff
@@ -21,22 +20,16 @@ export function pendingConfirmationsSelector(state: ConfirmMetamaskState) {
 export function pendingConfirmationsSortedSelector(
   state: ConfirmMetamaskState,
 ) {
-  return getPendingApprovals(state)
+  return Object.values(state.metamask.pendingApprovals ?? {})
```

Comment:
There's no value in doing this yet as we're not using `createSelector` so we'll always generate a new reference anyway, but at least we can re-use `getPendingApprovals`.